api/src/test/java/org/openmrs/validator/ConceptValidatorTest.java

```diff
@@ -57,7 +59,7 @@ public void validate_shouldFailIfThereIsADuplicateUnretiredConceptNameInTheLocal
 	public void validate_shouldFailIfTheObjectParameterIsNull() throws Exception {
 		Errors errors = new BindException(null, "concept");
 		new ConceptValidator().validate(null, errors);
-		Assert.assertTrue(errors.hasErrors());
+		Assert.assertTrue(errors.hasFieldErrors());
```

Comment:
Why are you changing this? If the object is null, then of course there would be no fields to validate so i would expect no filed errors